be/src/agent/task_worker_pool.cpp

```diff
@@ -618,6 +619,116 @@ void* TaskWorkerPool::_push_worker_thread_callback(void* arg_this) {
 
 void* TaskWorkerPool::_publish_version_worker_thread_callback(void* arg_this) {
     TaskWorkerPool* worker_pool_this = (TaskWorkerPool*)arg_this;
+
+    std::unique_ptr<ThreadPool> threadpool;
+    auto st = ThreadPoolBuilder("publish_version")
+                      .set_min_threads(config::partition_publish_version_worker_count)
+                      .set_max_threads(config::partition_publish_version_worker_count)
+                      .set_max_queue_size(32)
+                      .set_idle_timeout(MonoDelta::FromMilliseconds(2000))
+                      .build(&threadpool);
+
+    // Use a lambda to make code clearer.
+    auto publish_version_in_parallel = [&worker_pool_this, &threadpool](
```

Comment:
why not use function for so long lambda?

Reply:
Yeah, it's a pretty ugly lambda. The aim of lambda is to avoid adding a member function to the task_worker_pool. Is a friend function better? @DeepThinker666

Reply:
why to avoid adding a member function in task_worker_pool? I just do not like the function _publish_version_worker_thread_callback with so many code lines.

Reply:
OK. The lambda functions is already changed to a member function.